versions-common/src/main/java/org/codehaus/mojo/versions/api/ArtifactVersions.java

```diff
@@ -131,7 +130,7 @@ public boolean equals(Object o) {
     public int hashCode() {
         return new HashCodeBuilder(17, 37)
                 .append(getArtifact())
-                .append(getVersions())
+                .append(getVersions(true))
```

Comment:
Do we change default value?

Reply:
Same as above -- I'm not changing the default value.